### DISCUSSION TITLE
rustdesk-1.2.1-0 fedora28 != centos8 rhel8 loss of compatibility

### DISCUSSION THREAD

**Discussion by SergeyMy** [rustdesk/rustdesk#5005] [General]
loss of compatibility
//nightly builds 14-07-2023
`./rustdesk`
...
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: Invalid argument(s): Failed to load dynamic library 'librustdesk.so': /lib64/libm.so.6: version `**GLIBC_2.29**' not found

`/lib64/libc.so.6`
GNU C Library (GNU libc) stable release **version 2.28.**
Copyright (C) 2018 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.
Compiled by GNU CC version 8.5.0 20210514 (Red Hat 8.5.0-18).
libc ABIs: UNIQUE IFUNC ABSOLUTE
For bug reporting instructions, please see:
<http://www.gnu.org/software/libc/bugs.html>.

PS tested on Rocky Linux8 RHEL8 updates current as of 14-07-2023

**Comment by rustdesk** [comment]
Sadly, I will have to tell you 1.2 does not support centos 8 any more. I will remove centos from the file name.

**Reply by SergeyMy** [reply]
sorry and sadness, 1.2.1 builds perfectly from source on RHEL8(RL8)

**Comment by rustdesk** [comment]
We have to upgrade to build on Ubuntu 20.04 for Wayland support. Between Wayland and old distros, we chose Wayland, the future of Linux Desktop.

**Reply by SergeyMy** [reply]
Ubuntu deb branch, fedora rpm branch. abandoning RHEL8 is equal to the loss of a significant part of users

**Reply by rustdesk** [reply]
Do you think we do not know? But the world is moving forward, you have to give up something, including some users. But We will still try our most to help self-hosting Pro users for old Linux distro.

**Comment by rustdesk** [comment]
Try out the new nightly build https://github.com/rustdesk/rustdesk/releases/tag/nightly.

**Reply by SergeyMy** [reply]
yes it works correctly now
